CleverPush/CleverPush/Source/CPTopicsViewController.m

```diff
@@ -272,8 +279,10 @@ - (UIView *)tableView:(UITableView *)tableView viewForHeaderInSection:(NSInteger
     }
     
     UILabel* deselectEverything = [[UILabel alloc] init];
-    deselectEverything.text = [CPTranslate translate:@"deselectEverything"];
-    deselectEverything.frame = CGRectMake(10.0, (44 - switchSize.height) / 2.0f, tableView.bounds.size.width - (10 + switchSize.width), switchSize.height);
+    deselectEverything.text = @"Deselect Everything";
```

Comment:
Why `[CPTranslate translate:@"deselectEverything"]` removed?

Reply:
I was testing with static text that's why I was there and forget to keep the original one.
I will replace with original one.